### PR TITLE
chore: release 0.35.0

### PR DESCRIPTION
**Pull request by rflechtner**
## fixes https://github.com/KILTprotocol/ticket/issues/3119

Minor release to add support for the new dispatchAs extrinsic in CType and Public Credential fetching logic.
Also updates type augmentation for the upcoming runtime upgrade to 1.12.1, which additionally introduces changes to the balances pallet that can be considered breaking.
The changes also require an update to polkadot/api^10.7.3. The dependency manifests have been updated accordingly.

### CODE REVIEW

File: packages/augment-api/src/interfaces/augment-api-errors.ts [hunk 1]
```diff
@@ -43,35 +43,43 @@ declare module '@polkadot/api-base/types/errors' {
...
+       * Transfer/payment would kill account.
+       **/
+      Expendability: AugmentedError<ApiType>;
       /**
        * Balance too low to send value.
        **/
       InsufficientBalance: AugmentedError<ApiType>;
       /**
-       * Transfer/payment would kill account
+       * Account liquidity restrictions prevent withdrawal.
```
Comment (ntn-x2) [inline packages/augment-api/src/interfaces/augment-api-errors.ts:66]: Wouldnt' these be breaking changes that require a 0.35 version?

**Review by ntn-x2 — APPROVED** [review]
Approving in case you need to merge after I'm gone.

File: packages/augment-api/metadata/spiritnet.json
[the remark this answers is not in the document]
Comment (rflechtner) [reply packages/augment-api/metadata/spiritnet.json:1]: yup, 1.12.1

File: packages/augment-api/src/interfaces/augment-api-errors.ts (hunk 1 above)
Comment (rflechtner) [reply packages/augment-api/src/interfaces/augment-api-errors.ts:66]: I was thinking the same, yeah. Let's make it a new minor.

**Comment by rflechtner** [comment]
> We don't have a single test testing for the new stuff? Should we add it in the CType integration tests?

Added integration tests. For that I also had to update to the new standalone-node docker image.

**Pull request merged**